app/src/main/java/chat/rocket/android/layouthelper/chatroom/RoomListManager.java

```diff
@@ -126,6 +133,40 @@ private void onItemClick(View view) {
     }
   }
 
+  private void updateRoomsList() {
+    removeDeletedItem(unreadRoomsContainer, roomSubscriptionList);
+    removeDeletedItem(channelsContainer, roomSubscriptionList);
+    removeDeletedItem(dmContainer, roomSubscriptionList);
+
+    for (RoomSubscription roomSubscription : roomSubscriptionList) {
+      String name = roomSubscription.getName();
+      if (TextUtils.isEmpty(name)) {
+        continue;
+      }
+
+      String type = roomSubscription.getType();
+
+      if (unreadRoomMode && roomSubscription.getUnread() > 0) {
```

Comment:
`roomSubscription.isAlert()` would be better.

![slice](https://cloud.githubusercontent.com/assets/11763113/21287739/12ed6168-c4b6-11e6-9693-e6588ff5584c.png)